getting-started/dcos/linkerd-package-options.json

```diff
@@ -0,0 +1,13 @@
+{
+  "linkerd": {
+    "admin-port": 9990,
+    "config-filename": "linkerd.yaml",
+    "config-uri": "https://raw.githubusercontent.com/BuoyantIO/linkerd-examples/master/getting-started/dcos",
+    "cpus": 0.25,
+    "instances": 1,
+    "mem": 256.0,
+    "name": "linkerd-example",
+    "routing-port": 4140,
+    "resource-role": "slave_public"
```

Comment:
Adding `resource-roles` for linkerd:
https://github.com/mesosphere/universe/pull/547

You will need to change that line to:
```
"resource-roles": "\"slave_public\""
```

The escaped quotes are a bit awkward, but it was the only way I could find to support injecting arrays into a marathon template. This allows us to do things like:
```
"resource-roles": "\"slave_public\", \"*\""
```

Would you mind updating these files as well...?
https://github.com/BuoyantIO/linkerd-examples/blob/master/gob/dcos/namerd-package-options.json
https://github.com/BuoyantIO/linkerd-examples/blob/master/gob/dcos/README.md